package.json

```diff
@@ -79,10 +79,10 @@
   },
   "devDependencies": {
     "@edx/browserslist-config": "1.2.0",
-    "@edx/frontend-build": "13.0.5",
-    "@edx/react-unit-test-utils": "^1.7.0",
+    "@openedx/frontend-build": "13.0.27",
+    "@edx/react-unit-test-utils": "^2.0.0",
     "@edx/reactifex": "^1.0.3",
-    "@edx/stylelint-config-edx": "^2.3.0",
+    "@edx/stylelint-config-edx": "2.3.0",
```

Comment:
this was on `2.3.0` due to the `package-lock.json`, after rebuilding the package lock to cleanly handle the scope changes it updated to `2.3.3` and caused https://github.com/openedx/frontend-app-course-authoring/actions/runs/7746261151/job/21124075247 to fail with errors such as
```sh
src/index.scss
  1:9  ✖  Expected ""~@edx/brand/paragon/fonts"" to be "url("~@edx/brand/paragon/fonts")"                                                import-notation
```
Pinning it to `2.3.0` prevents those errors.

Reply:
No objections.  We can deal with that later.